src/socket/index.socket.js

```diff
@@ -1,4 +1,44 @@
-const initializeSocketIO = (io) => {};
+import ChatEventEnum from "../enums/chatEvents.enums";
+import isAuthorized from "../middlewares/auth.middleware";
+
+const initializeSocketIO = (io ) => {
+
+    // to check authToken from cookies
+    io.engine.use(isAuthorized);
```

Comment:
this will cause problem.
You need to add the user to socket not to the req as done in isAuthorized